### ISSUE TITLE
[layout] Align images on buttons

### ISSUE THREAD

**Issue by sir-indy** [espruino/BangleApps#1768]
### Affected hardware version

Bangle 2

### Your firmware version

2v13

### The bug

I'm using the layout library for my new timer app (the layout library is wonderful by the way, very easy to use!).

It was suggested (http://forum.espruino.com/comments/16499312/) to add images to the action buttons, rather than text. This makes internationalisation easier as there are less strings to translate, symbols are (or should be) universal.

I think an image alignment option should be added to the layout library, [here](https://github.com/espruino/BangleApps/blob/f2357e113e1626a816800a2a53ecc52736fbbb81/modules/Layout.js#L264), maybe by using some existing `halign` and `valign` options?:     
`if (l.src) g.setBgColor(bg).drawImage("function"==typeof l.src?l.src():l.src, l.x + 10 + (0|l.pad), l.y + 8 + (0|l.pad));`

Layout code used:
```
{
  type:"v", c: [
    {type:undefined, height:2},
    //{type:"txt", font:"15%", label:"TIMER", id:"title"},
    {type:"h", c: [
      {type:"v", c: [
        {type:"txt", font:"6x8", label:/*LANG*/"Hours", col:g.theme.fg2},
        {type:"img", pad:8, src:imgArrowUp, col:g.theme.fg2},
        {type:"txt", font:"20%", label:"0", id:"hours", filly:1, fillx:1},
        {type:"img", pad:8, src:imgArrowDown, col:g.theme.fg2}
      ]},
      {type:"v", c: [
        {type:"txt", font:"6x8", label:/*LANG*/"Minutes", col:g.theme.fg2},
        {type:"img", pad:8, src:imgArrowUp, col:g.theme.fg2},
        {type:"txt", font:"20%", label:"0", id:"mins", filly:1, fillx:1},
        {type:"img", pad:8, src:imgArrowDown, col:g.theme.fg2}
      ]},
      {type:"v", c: [
        {type:"txt", font:"6x8", label:/*LANG*/"Seconds", col:g.theme.fg2},
        {type:"img", pad:8, src:imgArrowUp, col:g.theme.fg2},
        {type:"txt", font:"20%", label:"0", id:"secs", filly:1, fillx:1},
        {type:"img", pad:8, src:imgArrowDown, col:g.theme.fg2}
      ]},
    ]},
    {type:"btn", src:imgPlay, id:"btnStart", fillx:1 }
  ], filly:1
};
```

![download (3)](https://user-images.githubusercontent.com/53864146/165733401-b118ff66-72c4-45bb-b8c0-bd7c8b35e40e.png)

**Comment by sir-indy** [comment]
I've made some mods to the layout library to fix this: #1788

**Comment by sir-indy** [comment]
Fixed in #1788!
